Keen.NetStandard/Keen.NetStandard.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Comment:
I think it's fine for this new binary to start of with the latest stable version. If we have a need, like a client's nuget deps, we can look at pushing it back.